realtime-trade-monitor/README.md

```diff
@@ -1,6 +1,6 @@
 # Real-time Trade Monitor
 
-A sample dashboard which uses [Hazelcast Demos](https://github.com/hazelcast/hazelcast-demos)
+A sample dashboard which uses [Hazelcast](https://github.com/hazelcast)
```

Comment:
This should be https://github.com/hazelcast/hazelcast

Reply:
corrected, obsolete dependencies have been removed